go/vt/callinfo/plugin_mysql.go

```diff
@@ -53,6 +54,18 @@ func (mci *mysqlCallInfoImpl) Text() string {
 	return fmt.Sprintf("%s@%s(Mysql)", mci.user, mci.remoteAddr)
 }
 
-func (mci *mysqlCallInfoImpl) HTML() template.HTML {
-	return template.HTML("<b>MySQL User:</b> " + mci.user + " <b>Remote Addr:<b> " + mci.remoteAddr)
+var mysqlTmpl = template.Must(template.New("tcs").Parse("<b>MySQL User:</b> {{.MySQLUSer>}}<b>Remote Addr:</b> {{.RemoteAddr}}"))
```

Comment:
Something in this is breaking the tests, but I can't spot it.
I notice that it is `USer`, not `User`, but that is consistent in the template and in the struct.

Reply:
nit: may as well change `MySQLUSer` to `MySQLUser`.

Reply:
@deepthi Already fixed!